### PR TITLE
Features Section of the new wechaty landing page (Section 2)

### PR DESCRIPTION
**Pull request by arnabiscreating**
Issue Number ( #912 )
This commit comprises of an atomic PR of Second section (Features Section with CSS) for the Wechaty landing page.
The green rectangle serves as a placeholder for the introduction video.

Images of features section running locally

(Desktop Version)
Screenshot 2021-09-08 at 1 16 16 PM

(Mobile Version)
Screenshot 2021-09-08 at 1 15 14 PM
Screenshot 2021-09-08 at 1 15 27 PM

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — COMMENTED** [review]
Check my comment on #1267 : )

**Comment by arnabiscreating** [comment]
Hello @Rohitesh-Kumar-Jain 
I've applied contrast to identify between distinct elements of the landing page, as per the recommended designs. The contrast is a little mild to match the overall UI, but each section is easily distinguishable. 
In addition, I updated the proposed changes and removed classnames and other style elements.

Here are some photographs from a locally run website that show contrast in sections.
Screenshot 2021-09-10 at 10 40 52 PM
Screenshot 2021-09-10 at 10 41 03 PM
Screenshot 2021-09-10 at 10 41 13 PM
Screenshot 2021-09-10 at 10 41 22 PM

File: docusaurus/src/pages/features-section/features-section.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            Why you would love to use this ?
+          </p>
+        </div>
+        <div id="featuresSectionUpperWrapper">
+          <div id="checkCircleWrapper">
+            <div id="checkCircle">
+              <img
+                alt="Check Circle Vector"
+                id="vector"
+                src="https://static.overlay-tech.com/assets/a633a787-7bc5-4b36-abe7-fbc1a8a48f1f.svg"
```
Comment (huan) [inline docusaurus/src/pages/features-section/features-section.tsx]: We need to save those images to local

**Review by huan — DISMISSED** [review]

**Review by proudofsimin — APPROVED** [review]

File: docusaurus/src/pages/features-section/features-section.css [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  overflow-y: hidden;
+  overflow-x: hidden;
+}
+
+#number100 {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/features-section/features-section.css]: if you may add some decription about this number100 that would be great

File: docusaurus/src/pages/features-section/features-section.css [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+  display: -ms-flexbox;
+  display: flex;
+  text-align: center;
+  margin-left: 0%;
+  letter-spacing: 4px;
+  margin-top: 10px;
+  letter-spacing: 0.1em;
+}
+
+#number100Two {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/features-section/features-section.css]: same as above

File: docusaurus/src/pages/features-section/features-section.css [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  overflow-y: hidden;
+  overflow-x: hidden;
+}
+
+#number10000 {
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/features-section/features-section.css]: same as above

File: docusaurus/src/pages/features-section/features-section.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          <div id="featuresSectionRectangle" />
+          <p id="featuresSectionHeadline">Why you would love to use this ?</p>
+        </div>
+        <div id="featuresSectionUpperWrapper">
+          <div id="checkCircleWrapper">
+            <div id="checkCircle">
+              <img alt="Check Circle Vector" id="vector" src="tick-mark.webp" />
+            </div>
+            <div id="checkCircle">
+              <img alt="Check Circle Vector" id="vector" src="tick-mark.webp" />
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/src/pages/features-section/features-section.tsx]: Thanks for automatically implementing suggestion across the PRs : )
Great work 🎉

**Review by Rohitesh-Kumar-Jain — CHANGES_REQUESTED** [review]

File: docusaurus/src/pages/features-section/features-section.css (hunk 4 above)
Comment (arnabiscreating) [reply docusaurus/src/pages/features-section/features-section.css]: Hello @Rohitesh-Kumar-Jain 
Okay, to make them more descriptive, I'll write a comment.

File: docusaurus/src/pages/features-section/features-section.tsx (hunk 5 above)
Comment (arnabiscreating) [reply docusaurus/src/pages/features-section/features-section.tsx]: Thank You : )

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Thanks for using more descriptive name, when the names aren't descriptive enough add comments

**Review by sajen-k — APPROVED** [review]
LGTM

**Pull request merged**